modules/installation-machines-bare-metal.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-machines-bare-metal-{context}"]
+= Creating the cluster
+
+To create the {product-title} cluster, you provision machines by using the Ingition config
+files that you generated with the installation program.
+
+.Prerequisites
+
+* Your machines must have internet access.
+* Create the required infrastructure for the cluster.
+* You obtained the installation program and generated the Ignition config files
+for your cluster.
+
+.Procedure
+
+. Obtain the {op-system-first} images.
+
+. Use the the ignition config files that you generated by using the installation
+program to create the required bootstrap, worker, and master nodes.
+
+. Monitor for bootstrap-complete
+to monitor cluster bootstrapping.
++
+----
+$ openshift-install --dir test-bare-metal upi bootstrap-complete
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
+INFO API v1.12.4+c53f462 up
+INFO Waiting up to 30m0s for the bootstrap-complete event...
+----
++
+The command succeeds when the Kubernetes APIServer signals that it has been
+bootstrapped on the control plane machines.
+
+. Confirm that the cluster recognizes the machines:
++
+----
+$ oc get nodes
+
+NAME      STATUS    ROLES   AGE  VERSION
+master-0  NotReady  master  50s  v1.12.4+c53f462
+master-1  NotReady  master  63s  v1.12.4+c53f462
+master-2  NotReady  master  64s  v1.12.4+c53f462
+worker-0  NotReady  worker  76s  v1.12.4+c53f462
+worker-1  NotReady  worker  70s  v1.12.4+c53f462
+worker-2  NotReady  worker  69s   v1.12.4+c53f462
+----
++
+The output lists all of the machines that you created.
+
+. Approve all the pending certificate signing requests:
```

Comment:
cc @sjenning since machine-approver is not going to approve the server certificates for kubelet on bare-metal. Do you have docs that point to how / which CSRs users must approve or any other documentation.

Reply:
@sjenning +1. I saw in your video that you just approved all of the CSRs, but if that's not what we want users to do in production, I'll update it.

Reply:
I think we want to loosely encourage review before blanket approval but prescribing a precise method for review is problematic. Perhaps something like the following

_Review the pending CSRs to ensure they match the expected hosts joining the cluster,  `oc get csr`
Approve them either individually or in bulk using a command like the following_

If the CSR approval is still two step (client, then server) we also need to make sure that we approve the server side too, i have no idea if that's still the case.

Reply:
@sdodson, I've updated the text about the CSRs. Please let me know if you have more suggestions!